### ISSUE TITLE
Editor's guide for images mentions non-existent fields

### ISSUE THREAD

**Issue by gasman** [wagtail/wagtail#6375]
From https://docs.wagtail.io/en/v2.10/editor_manual/documents_images_snippets/images.html:

> Clicking an image will allow you to edit the data associated with it. This includes the Alt text, the photographers credit, the medium of the subject matter and much more.
> Changing the alt text here will alter it for all occurrences of the image in carousels, but not in inline images, where the alt text can be set separately.

The basic image model in an out-of-the-box Wagtail installation does not have fields for alt text, credit or medium, and carousels are not a core Wagtail concept, so how they behave is site-specific. (The inline model used for a carousel could, and probably should, have its own context-specific alt text field...) This text is presumably a hangover from the original Wagtail site built for [RCA](https://www.rca.ac.uk/), (which implements these fields with a [custom image model](https://docs.wagtail.io/en/stable/advanced_topics/images/custom_image_model.html)) and should be rewritten or removed.
